plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergModule.java

```diff
@@ -48,8 +49,8 @@
     public void configure(Binder binder)
     {
         binder.bind(IcebergTransactionManager.class).in(Scopes.SINGLETON);
+        binder.bind(Key.get(boolean.class, TranslateHiveViews.class)).toInstance(false);
```

Comment:
If this is Thrift specific, it can go in `IcebergHiveMetastoreCatalogModule`

Reply:
i see thjis added to `IcebergHiveMetastoreCatalogModule.java` but not removed here